src/ui/components/address-action/SingleAsset/SingleAsset.tsx

```diff
@@ -97,18 +95,7 @@ function FungibleAsset({
                 )}{' '}
                 <AssetLink asset={fungible} address={address} />
               </UIText>
-              {actionType === 'approve' &&
-              Boolean(quantity) &&
-              allowanceViewHref ? (
-                <UIText
-                  as={TextLink}
-                  kind="small/accent"
-                  style={{ color: 'var(--primary)' }}
-                  to={allowanceViewHref}
-                >
-                  Edit
-                </UIText>
-              ) : null}
+              <RenderArea name="single-asset-quantity-right" />
```

Comment:
So we don't have to pass the `allowanceViewHref` down to the `SingleAsset` component anymore